qeb-hwt-github-app/overlays/ocp/configmap.yaml

```diff
@@ -6,11 +6,3 @@ metadata:
 data:
   subdomain: qeb-hwt
   webhook-callback-url: "http://qeb-hwt-webhook-receiver-aicoe-prod-bots.apps.ocp.prod.psi.redhat.com"
----
-# Duplicate configmap as this would be moved to backend namespace
```

Comment:
if we are removing this, then we should remove  it from kustomize config.
https://github.com/thoth-station/thoth-application/blob/master/qeb-hwt-github-app/overlays/ocp/kustomization.yaml#L46-L51

Reply:
Removed! Thanks :)